dhis-2/dhis-support/dhis-support-external/src/main/java/org/hisp/dhis/external/conf/ConfigurationKey.java

```diff
@@ -667,7 +667,10 @@ public enum ConfigurationKey {
   SWITCH_USER_ALLOW_LISTED_IPS(
       "switch_user_allow_listed_ips", "localhost,127.0.0.1,[0:0:0:0:0:0:0:1]", false),
 
-  MAX_FILE_UPLOAD_SIZE_BYTES("max.file_upload_size", Integer.toString(10_000_000), false);
+  MAX_FILE_UPLOAD_SIZE_BYTES("max.file_upload_size", Integer.toString(10_000_000), false),
+
+  /** CSRF feature. Enable or disable the feature. */
+  CSRF_ENABLED("csrf.enabled", Constants.OFF, true);
```

Comment:
Should we use a more namespaced name like `http.security.csrf.enabled` ? or something like it - maybe the other configurations we have in that area already establish a common prefix?

Reply:
I like that, will use that. Better late than never, we haven't really done that before, but why not start.